src/components/UI/Text/URL.vue

```diff
@@ -3,19 +3,30 @@
     <!-- URL link -->
     <p class="text-md lg:text-lg underline h-full place-self-center">{{ link }}</p>
     <!-- copy button -->
-    <button @click="copyToClipboard()">
+    <button @click="copyToClipboard()" v-tooltip="'Copy link'">
       <img src="@/assets/images/copy.svg" class="w-10 h-10 min-width-full" />
     </button>
+    <toast ref="toast"></toast>
   </div>
 </template>
 
 <script>
+import Toast from "@/components/UI/Message/Toast.vue";
+
 export default {
+  data() {
+    return {
+      toastLife: 1000,
```

Comment:
```suggestion
      toastLife: 3000,
```
1 seconds feels very small